scripts/check-users-in-space-developer-role.ps1

```diff
@@ -1,37 +1,81 @@
 #!/usr/bin/env pwsh
 
-param([String] $space, [String] $slackwebhook)
-
-$data = @()
+param($space_name, $slackwebhook, $delete_arg, $protected_usernames_csv)
 
+$ErrorActionPreference = "Stop"
 $bearer = $(cf oauth-token)
 
 $headers = @{
   Authorization = "$bearer"
   Accept        = "application/json"
 }
 
-$get_space_details = Invoke-RestMethod "https://api.london.cloud.service.gov.uk/v3/spaces?names=$space"  -H $headers
-$space_guid = $get_space_details.resources[0].guid
+$DELETE = $delete_arg -eq "delete"
+if ($DELETE) {
```

Comment:
remove is a better term than delete in this case?

Reply:
why not `if ($delete_arg -eq "delete")`............? and remove line 13?

Reply:
The variable is used again for the logic later on

Reply:
@vigneshmsft I've changed this parameter to `unset` and forced using named parameters for clarity